CONTRIBUTING.md

```diff
@@ -177,6 +177,24 @@ python3-foobar-pip:
       packages: [foobar]
 ```
 
+In constrast to normal python entries, which are often different for python 2 and 3, pip entries for python 2 and 3 are almost always identical.
+Hence no new entry would be needed. Though this would leave us with a mess of `python3-*`, `python-*-pip` and `python3-*-pip` entries.
+To prevent this, the `python3-*-pip` entry should be mapped to the legacy `python-*-pip` entry by using yaml anchors and aliases.
+(Preferably this was the other way around. So the `python3-*-pip` entry containing the contents and the anchor and the legacy `python-*-pip` entry being aliased to it.
+Though the anchor should be defined before the anchor is used and `python3-` entries come after `python-` entries in alphabetical order.)
+
+For example:
+
+```yaml
+python-foobar-pip: &migrate_eol_2025_04_30_python3_foobar_pip # Anchor
```

Comment:
why the anchor needs to have a date and a `migrate_eol` prefix?
can't the anchor just be the name of the key? e.g. `&python3_foobar_pip`

Reply:
This is suggested by @tfoote in https://github.com/ros/rosdistro/issues/28891

Reply:
Okey, that makes sense.
Information about how to pick the EOL date should be added here though.

Reply:
Resolved by ab459b9